src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import seedu.address.MainApp;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+import seedu.address.storage.AddressBookStorage;
+import seedu.address.storage.JsonAddressBookStorage;
+import seedu.address.storage.StorageManager;
+
+/**
+ * Exports records to a text file.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Exports to text file in the \"data\" folder \n"
+            + "Parameters: FILENAME\n"
+            + "Example: " + COMMAND_WORD + " records1.json";
+
+    public static final String MESSAGE_SUCCESS = "Exported the records!";
+    private static final String MESSAGE_FAILURE = "Problem while writing to the file.";
```

Comment:
Could look into why one of these constants is public while the other is kept private